ui/espresso/ScreenshotSample/app/src/androidTest/java/com/example/android/testing/espresso/screenshotsample/ScreenshotTest.kt

```diff
@@ -0,0 +1,68 @@
+package com.example.android.testing.espresso.screenshotsample
+
+
+//import androidx.test.core.view.captureToBitmap
+
+import androidx.test.core.app.takeScreenshot
+import androidx.test.core.graphics.writeToTestStorage
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.espresso.screenshot.captureToBitmap
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestName
+import org.junit.runner.RunWith
+import java.io.IOException
+
+/*
+ * Illustrates usage of APIs to capture a bitmap from view and saving it to test storage.
+ */
+@RunWith(AndroidJUnit4::class)
+class ScreenshotTest {
+
+  // a handy JUnit rule that stores the method name, so it can be used to generate unique
+  // screenshot files per test method
+  @get:Rule
+  var nameRule = TestName()
+
+  @get:Rule
+  val activityScenarioRule = activityScenarioRule<MainActivity>()
+
+  /**
+   * Captures and saves an image of the entire [MainActivity] contents.
+   */
+  @Test
+  @Throws(IOException::class)
+  fun saveActivityBitmap() {
+    onView(isRoot())
+      .captureToBitmap()
+      .writeToTestStorage(nameRule.methodName)
```

Comment:
Might want to mention that the file is not automatically pulled from the device yet?

Reply:
Looks like the header is not updated? I didn't see there's a new revision. Did you forget to upload?